scripts/brewkit/python-venv.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/sh
+
+CMD_NAME=$1
+PROJECT_NAME=$2
+VERSION=$3
+PREFIX=$4
+PYTHON_VERSION=$5
+PYTHON_VERSION_MAJ=$6
+PYTHON_VERSION_MIN=$7
```

Comment:
Oof, well yeah variables is clearer. But I believe we can streamline this to just two vars, I'll do that after merge.

Still want to see meson build switched to using this so we can see it working though. You can do that in this PR.

Reply:
Ok, I added that.